test/cfg/qt.cpp

```diff
@@ -775,3 +775,13 @@ void unusedVariable_qtContainers() // #10689
     QLatin1String ql1s;
 }
 
+void unreadVariable_QMapIterator(QMap<QString, QObject*>& m)
+{
+    auto it = m.find("abc"); // #12662
+    if (it != m.end()) {
+        // cppcheck-suppress checkLibraryFunction // TODO
```

Comment:
Please add a ticket about this if it is not yet tracked (I assume it already is).

Reply:
https://trac.cppcheck.net/ticket/12673